### DISCUSSION TITLE
Admin Panel is not working propperly. Items are missing

### DISCUSSION THREAD

**Discussion by HardySeitz** [microweber/microweber#1001] [Q&A]
After trying an update im Microweber I get this Layout. 
Most of the Buttons do have no action.
How can I repair this Installation?
![Bild 23 02 23 um 19 03](https://user-images.githubusercontent.com/126193593/220992188-2a9fc11c-875e-4d64-bbcb-10464b12cc72.jpg)

**Comment by markcjf** [comment]
I had a similar issue for which I got no reply - my issue was that after creating some additional pages part of the admin area would not work (500 error). My solution was to roll back to an earlier backup but I had to recreate the pages and all the information contained therein.
Have to say this forum is pretty much the worse I have come across, assistance is next to none.
